### PR TITLE
Feature/c

### PR DESCRIPTION
**Pull request by KenYuChang**
做好reply model的兩個API，getTweetReply, postTweetReply，並通過測試！

### CODE REVIEW

File: test/requests/reply.spec.js
Comment (Jasmineeds) [inline test/requests/reply.spec.js:1]: Suggest avoid changing in test files.

File: routes/index.js [hunk 1]
```diff
@@ -46,23 +76,26 @@ router.post(
...
   authenticated,
   authenticatedUser,
-  userController.getFollowings
+  replyController.postTweetReply
 );
+
+//tweets
 router.get(
-  "/api/users/:id/followers",
+  "/api/tweets/:tweet_id",
```
Comment (Jasmineeds) [inline routes/index.js:95]: Kindly adopt original plan for routes

File: controllers/user-controller.js [hunk 2]
```diff
@@ -142,6 +142,7 @@ const userController = {
         if (followings.Followings.length === 0)
           return res.status(200).json({ isEmpty: true });
         const followingId = getUser(req).Followings.map((user) => user.id);
+        console.log(followingId);
```
Comment (Jasmineeds) [inline controllers/user-controller.js:145]: Suggest omit console.log() in master doc.

File: controllers/reply-controller.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            name: reply["User.name"],
+            avatar: reply["User.avatar"],
+          },
+          // tweets
+          tweetData: {
+            tweetId: reply.TweetId,
+            tweetOwnerId: reply["Tweet.User.id"],
+            tweetOwnerAccount: reply["Tweet.User.account"],
+            createdAt: new Date(reply.createdAt).toLocaleString(),
+          },
```
Comment (Jasmineeds) [inline controllers/reply-controller.js:45]: Great way to formate data for front-end.

File: controllers/reply-controller.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    const getTweetId = req.params.id;
+    const userId = getUser(req).id;
+    const { comment } = req.body;
+
+    return Tweet.findByPk(getTweetId)
+      .then((tweet) => {
+        if (!tweet) throw new Error("Tweet not exist!");
+        return Reply.create({
+          comment,
+          tweetId: getTweetId,
```
Comment (Jasmineeds) [inline controllers/reply-controller.js:61]: Will it be better using as follow?
`const tweetId = req.params.id`

**Review by Jasmineeds — CHANGES_REQUESTED** [review]
Kind view the comments, thanks!

File: test/requests/reply.spec.js
Comment (KenYuChang) [reply test/requests/reply.spec.js:1]: prettieir did this for me.

**Review by Jasmineeds — APPROVED** [review]
Good!

**Pull request merged**